cli/simulate_pixels.py

```diff
@@ -333,7 +366,7 @@ def run_simulation(input_filename,
 
             RangePush("sum_pixels_signals")
             # Here we combine the induced current on the same pixels by different tracks
-            TPB = (8,8,8)
+            TPB = (1,1,64)
```

Comment:
This is because it's faster?

Reply:
I guess - this was not really intended in this PR. I think you maybe mentioned it a while ago, and I tried running with these parameters. I didn't notice a difference, but yeah this was an accident.